howso/direct/client.py

```diff
@@ -435,56 +437,62 @@ def check_name_valid_for_save(
 
     def create_trainee(  # noqa: C901
         self,
-        trainee: Trainee,
+        name: Optional[str] = None,
+        features: Optional[SingleTableFeatureAttributes] = None,
         *,
-        library_type: Optional[Literal["st", "mt"]] = None,
-        max_wait_time: Optional[Union[int, float]] = None,
         overwrite_trainee: bool = False,
-        resources: Optional[Union[TraineeResources, Dict]] = None,
-    ) -> Trainee:
+        persistence: Persistence = "allow",
+        id: Optional[str] = None,
+        library_type: Optional[Library] = None,
+        max_wait_time: Optional[Union[int, float]] = None,
+        metadata: Optional[MutableMapping[str, Any]] = None,
+        project: Optional[Union[str, Dict]] = None,
+        resources: Optional[MutableMapping[str, Any]] = None,
+        client: Optional[AbstractHowsoClient] = None
```

Comment:
Client should not be a parameter to a client method.

Reply:
Addressed by: 5717250